ray_ci/test_pipeline_ci.py

```diff
@@ -193,5 +196,32 @@ def test_pipeline_update_queue():
     os.environ.pop("RUNNER_QUEUE_SMALL", None)
 
 
+def test_read_lineline():
+    with tempfile.TemporaryDirectory() as tmpdir:
+        pipeline_path = Path(os.path.join(tmpdir, "pipeline.yml"))
+        with open(pipeline_path, "w") as f:
+            f.write("#ci:group=foo")
```

Comment:
one more test where there is a group and steps is not empty?

Reply:
done